rotkehlchen/tests/unit/test_price_historian.py

```diff
@@ -142,3 +163,38 @@ def test_token_to_fiat_via_second_oracle(fake_price_historian):
     assert price == expected_price
     for oracle_instance in price_historian._oracle_instances[0:2]:
         assert oracle_instance.query_historical_price.call_count == 1
+
+
+def test_manual_oracle_correctly_returns_price(globaldb, manual_historian):
+    """Test that the manual oracle correctly returns price for asset
+    """
+    price_historian = manual_historian
+    # Add price at timestamp
+    globaldb.add_single_historical_price(
+        HistoricalPrice(
+            from_asset=A_BTC,
+            to_asset=A_USD,
+            price=30000,
+            timestamp=Timestamp(1611595466),
+            source=HistoricalPriceOracle.MANUAL,
+        ),
+    )
+    # Make the other oracles fail
+    expected_price = Price(FVal('30000'))
+    oracle_instances = price_historian._oracle_instances
+    oracle_instances[0].query_historical_price.side_effect = PriceQueryUnsupportedAsset('bitcoin')
+    oracle_instances[1].query_historical_price.side_effect = PriceQueryUnsupportedAsset('bitcoin')
+    # Query price, should return the manual price
+    price = price_historian.query_historical_price(
+        from_asset=A_BTC,
+        to_asset=A_USD,
+        timestamp=Timestamp(1611595466),
```

Comment:
Give a timestamp a bit above or below to see the range we give at the oracle works.